.github/workflows/test_branches.yml

```diff
@@ -654,14 +653,19 @@ jobs:
 
     - name: Install Pyomo
       run: |
+        # Turn on cython through env vars
+        if [[ "${{ matrix.other }}" == *cython* ]]; then
+            echo "Cython requested; turning on cythonization options in environment"
+            export PYOMO_SETUP_ARGS=--with-cython
+        fi
```

Comment:
Why not just leave the `setup_options` matrix variable and then (unconditionally) add it to the `PYOMO_SETUP_ARGS` environment variable here?

Reply:
Because the only reason we use `setup_options` is for cythonization. I don't really care either way, but it seemed redundant, so tell me what you'd prefer.

Reply:
In general, I would prefer to *not* key job behavior off the job name (this brings back bad memories of codes where the input deck filename could set some of the program arguments).  It seems fragile and not extensible.  (so yeah - if you are asking about my *preference*, it would be to have an explicit argument in the matrix definition.  :) )